src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+function compareFunction(current, next) {
+  const firstValue = String(current);
+  const secondValue = String(next);
+
+  if (firstValue > secondValue) {
+    return 1;
+  } else if (firstValue === secondValue) {
+    return 0;
+  }
```

Comment:
```suggestion
  }
  
  if (firstValue === secondValue) {
    return 0;
  }
```
it's better to avoid `else if` - use common `if` instead if it's possible (when you return value for example, as you do here)